server/protocols/jmap/src/main/java/org/apache/james/jmap/JMAPUrls.java

```diff
@@ -23,4 +23,5 @@
 
     String JMAP = "/jmap";
     String AUTHENTICATION = "/authentication";
+    String DOWNLOAD = "/download/*";
```

Comment:
1/ Right for the star
2/ It's not has easy with servlets: 
  we should add a star at the end of the URL which binds the servlet, 
  then we have to check inside the servlet that HttpServletRequest.getPathInfo() contains the blobId.